docs/mdbook/src/android/annotation-guide.md

```diff
@@ -0,0 +1,88 @@
+# Annotation: Marker
+
+This guide will show you how to add Markers in the map.
+
+`Annotation` is an overlay on top of a Map. In package
+`com.mapbox.mapboxsdk.annotations`, it has the following subclasses:
+1. [Marker]
+2. [Polyline]
+3. [Polygon]
+
+A Marker shows an icon image at a geographical location. By default, marker uses
+a [provided image] as its icon.
+
+![marker image]
+
+Or, the icon can be customized using [IconFactory] to generate an
+[Icon] using a provided image.
+
+For more customization, please read the documentation about [MarkerOptions].
+
+In this showcase, we continue the code from the [Quickstart],
+rename Activity into `JsonApiActivity`,
+and pull the GeoJSON data from a free and public API.
+Then add markers to the map with GeoJSON:
+
+1. In your module Gradle file (usually `<project>/<app-module>/build.gradle`), add
+   `okhttp` to simplify code for making HTTP requests.
+
+
+    ```gradle
+    dependencies {
+        ...
+        implementation 'com.squareup.okhttp3:okhttp:4.10.0'
+        ...
+    }
+    ```
+
+2. Sync your Android project the with Gradle files.
+
+3. In `JsonApiActivity` we add a new variable for `MapboxMap`.
+   It is used to add annotations to the map instance.
+
+```kotlin
+{{#include ../../../../platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/annotation/JsonApiActivity.kt:top}}
+```
+
+4. Rewrite `mapview.getMapSync()`. Now we assign values for `maplibreMap`.
```

Comment:
I would say Call instead of Rewrite.

Reply:
How about [d0c66a8](https://github.com/maplibre/maplibre-native/pull/1135/commits/d0c66a8de81129857535b15aabc257519fd96c40)?

```diff
-4. Rewrite `mapview.getMapSync()`. Now we assign values for `maplibreMap`.
-   Also, call the `getEarthQuakeDataFromUSGS()` method to make a HTTP request.
+4. Call `mapview.getMapSync()` in order to get a `MapboxMap` object.
+   After `mapboxMap` is assigned, call the `getEarthQuakeDataFromUSGS()` method
+   to make a HTTP request and transform data into the map annotations.
```